### PR TITLE
feat: add PoTeC dataset

### PR DESCRIPTION
**Pull request by prassepaul**
## Description

This PR adds the PoTeC dataset to pymovements.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: docs/source/bibliography.bib [hunk 1]
```diff
@@ -35,6 +35,14 @@ @misc{JuDo1000
     doi = {10.17605/OSF.IO/5ZPVK},
 }
 
+@misc{potec,
+    author = {Jäger, Lena A. and Haller, Patrick and Jakobi, Deborah Noemie},
+    title = {{Potsdam Textbook Corpus}},
+    year = {2021},
+    pages = {},
+    doi = {10.17605/OSF.IO/DN5HP},
+}
```
[the remark this answers is not in the document]
Comment (SiQube) [reply docs/source/bibliography.bib:44]: okay then we should add an issue after this PR is merged

File: src/pymovements/datasets/potec.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+from pymovements.dataset.dataset_library import register_dataset
+from pymovements.gaze.experiment import Experiment
+
+
+@dataclass
+@register_dataset
+class PoTeC(DatasetDefinition):
+    """PoTeC dataset :cite:p:`potec`.
+
+    The Potsdam Textbook Corpus (PoTeC) is a corpus of eye-tracking-while-reading data where
```
Comment (dkrako) [inline src/pymovements/datasets/potec.py:39]: can you add which eye is recorded to the docstring?

File: src/pymovements/datasets/potec.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""This module provides an interface to the JuDo1000 dataset."""
```
Comment (dkrako) [inline src/pymovements/datasets/potec.py:20]: copy pasta

**Review by dkrako — APPROVED** [review]
Somehow the dataset ist not part of `DatasetLibrary` and I don't get why.

- [ ] Please add the respective test cases to this file:
https://github.com/aeye-lab/pymovements/blob/main/tests/unit/datasets/datasets_test.py

- [ ] make sure that the integration test passes

You can run a single test case like this:
```bash
pytest tests/integration/public_dataset_processing_test.py::test_public_dataset_processing[PoTeC]
```

**Review by dkrako — CHANGES_REQUESTED** [review]
haha not so fast, I clicked the wrong box, Reguest changes it is!

File: src/pymovements/datasets/potec.py (hunk 3 above)
Comment (prassepaul) [reply src/pymovements/datasets/potec.py:20]: done

**Comment by prassepaul** [comment]
> Somehow the dataset ist not part of `DatasetLibrary` and I don't get why.
> 
> * [x]  Please add the respective test cases to this file:
>   https://github.com/aeye-lab/pymovements/blob/main/tests/unit/datasets/datasets_test.py
> * [x]  make sure that the integration test passes
> 
> You can run a single test case like this:
> 
> ```shell
> pytest tests/integration/public_dataset_processing_test.py::test_public_dataset_processing[PoTeC]
> ```

To perform the pytest, you need to install pymovements from path....

**Pull request merged**